### PR TITLE
feat : Align analytics properties for apps and device models.

### PR DESCRIPTION
**Pull request by udayasri**
- Change `appType` to always send type : `app`
- Add new optional parameter `appVersion` that can be passed when configuring the player.

### CODE REVIEW

**Comment by udayasri** [comment]
Screenshot 2024-02-06 at 14 33 08

File: Sources/iOSClientExposurePlayback/Events/DeviceInfo.swift [hunk 1]
```diff
@@ -37,11 +37,13 @@ internal struct DeviceInfo {
...
+    
     internal var cdnInfo: CDNInfoFromEntitlement?
     
     internal var analyticsInfo: AnalyticsFromEntitlement?
     
-    internal init(timestamp: Int64, connection: String, type: String? = nil, tech: String, techVersion: String,  cdnInfo: CDNInfoFromEntitlement? = nil , analyticsInfo: AnalyticsFromEntitlement? = nil, appName: String? = nil ) {
+    internal init(timestamp: Int64, connection: String, type: String? = nil, tech: String, techVersion: String,  cdnInfo: CDNInfoFromEntitlement? = nil , analyticsInfo: AnalyticsFromEntitlement? = nil, appName: String? = nil, appVersion: String? = nil) {
         self.timestamp = timestamp
         self.connection = connection
         self.type = type
```
Comment (shobaki) [inline Sources/iOSClientExposurePlayback/Events/DeviceInfo.swift:49]: what is `type` here? and where is `appType`? 
You seem to always include these properties in other events, but not all of the in this event (DeviceInfo):
```
        var json: [String: Any] = [
            JSONKeys.eventType.rawValue: eventType,
            JSONKeys.timestamp.rawValue: timestamp,
            JSONKeys.connection.rawValue: connection,
            JSONKeys.player.rawValue: player,
            JSONKeys.deviceId.rawValue: device.deviceId,
            JSONKeys.deviceModel.rawValue: device.model,
            JSONKeys.os.rawValue: device.os,
            JSONKeys.appType.rawValue: "app",
            JSONKeys.osVersion.rawValue: device.osVersion,
            JSONKeys.manufacturer.rawValue: device.manufacturer,
            JSONKeys.height.rawValue: device.height,
            JSONKeys.width.rawValue: device.width
        ]
```

File: Sources/iOSClientExposurePlayback/Events/DeviceInfo.swift [hunk 2]
```diff
@@ -76,7 +79,7 @@ extension DeviceInfo: AnalyticsEvent {
             JSONKeys.deviceId.rawValue: device.deviceId,
             JSONKeys.deviceModel.rawValue: device.model,
             JSONKeys.os.rawValue: device.os,
-            JSONKeys.appType.rawValue: device.os,
+            JSONKeys.appType.rawValue: "app",
```
Comment (udayasri) [inline Sources/iOSClientExposurePlayback/Events/DeviceInfo.swift:82]: @shobaki  Here you have the `appType`

File: Sources/iOSClientExposurePlayback/Events/DeviceInfo.swift (hunk 1 above)
Comment (udayasri) [reply Sources/iOSClientExposurePlayback/Events/DeviceInfo.swift:49]: Type is for `Airplay`

```
internal let type: String?` 
/// Optional string indicating the nature of playback from this device. Can be used to indicate this is an `Airplay` session : ´Airplay´
```

Comment (shobaki) [reply Sources/iOSClientExposurePlayback/Events/DeviceInfo.swift:49]: Yes, I remember now as we discussed it on slack. It was removed in the new specification, but leave it for now to keep this PR isolated to the jira ticket.

**Review by iamramanjot — APPROVED** [review]
LGTM

**Pull request merged**